modules/apps/27-interchain-accounts/controller/types/params_test.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func TestValidateParams(t *testing.T) {
+	t.Parallel()
```

Comment:
This change can be made outside this pr?

Reply:
Man that shouldn't be there at all, sorry about that I removed it from the one that was targeting main but seem to have missed it here